WORKING_GROUPS.md

```diff
@@ -0,0 +1,213 @@
+  - [Documentation, Education and Websites](#documentation-education-and-websites)
+**Ongoing Efforts to Support Specific Uses:**
+  - [Integration with Web Browsers](#integration-with-web-browsers)
+  - [Dynamic Data and Capabilities](#dynamic-data-and-capabilities)
+  - [IPFS Cluster](#ipfs-cluster)
+
+#### JavaScript IPFS implementation
+
+- **Coordination**: https://github.com/ipfs/pm/blob/master/JS_CORE_DEV_MGMT.md
+- **Captain**: [David Dias](https://github.com/diasdavid/)
+
+Develop js-ipfs.
+
+**Responsibilities include**:
+- Create the Browser implementation of the IPFS Protocol.
+
+#### Golang IPFS implementation
+
+- **Coordination**: https://github.com/ipfs/pm/blob/master/GOLANG_CORE_DEV_MGMT.md
+- **Captain**: [Jeromy Johnson](https://github.com/whyrusleeping)
+
+Develop go-ipfs.
+
+**Responsibilities include**:
+- Create the Server and reference implementation of the IPFS Protocol.
+
+#### IPFS GUI
+
+- **Coordination**: https://github.com/ipfs/ipfs-gui
+- **Captain**: [Oli Evans](https://github.com/olizilla)
+
+Making IPFS GUIs simple, accessible, reusable and beautiful.
+
+**Responsibilities include**:
+- Fight complexity at every step.
+- IPFS must be usable and comprehensible for everyone.
+- Publish and promote this work. Make doing the right thing the easiest thing.
+- Demonstrate the nature of the system with effortless, coherent, and compelling interfaces.
+
+#### IPFS Infrastructure
+
+- **Coordination**: http://github.com/ipfs/infrastructure
+- **Captain**: [Erin Fahy](https://github.com/eefahy)
+
+Tools and systems for the IPFS community.
+
+**Responsibilities include**:
+- Manage and maintain the IPFS HTTP Gateway
+- Manage and maintain the IPFS Bootstrapper nodes
+- Manage and maintain the IPFS pinning service, pinbot
+- Monitor services and hosts used by the IPFS Dev teams
+- Provide wisdom to other users that want to host IPFS nodes
+
+#### Dev Team Enablement (QA, Testing, Support)
+
+- **Coordination**: https://github.com/ipfs/testing
+- **Captain**: [Victor Bjelkholm](https://github.com/VictorBjelkholm)
+
+The QA, Testing and Dev Team Enablement Working Group focuses on building developer tools, CI and automated maintenance tasks to improve the Developer Experience of the many IPFS (and related) projects.
+
+**Responsibilities include**:
+
+- Enable developers to develop faster by reducing friction from dev.
+- Upgrade and build dev tools (i.e AEgir and gx).
+- Create a sound testing infrastructure for all the IPFS projects.
+- Set the guidelines for writing, maintaining and managing codebases, test suites and contributions.
+- Define process and help achieve high quality Sustainable Open Source Modules.
+- Automating tasks for things such as: release cycle, ensuring code quality, activity reports.
+
+#### Community, Events & Evangelism
+
+> This Working Group hasn't been formed yet.
+
+- **Coordination**: https://github.com/ipfs/community
+- **Captain**: ??? 
```

Comment:
I wasn't sure if I would leave this one around or not. The truth is that there is a lot of work done and planned, but we don't have an official Captain for this front.

@mikeal given that you are the person leading most of the new charges, would you like to take this role and plan OKRs for next quarter? Knowing that this is a role that you are also actively looking to fill/replace yourself?